pkg/drivers/jetstream/jetstream.go

```diff
@@ -47,11 +47,12 @@ type JSValue struct {
 }
 
 // New get the JetStream Backend, establish connection to NATS JetStream. At the moment nats.go does not have
-// connection string support so kine will use nats://(token|username:password)hostname:port?bucket=bucketName&context=nats-context`
+// connection string support so kine will use nats://(token|username:password)hostname:port?bucket=bucketName&contextFile=nats-context`.
+// If contextFile is provided then do not provide a hostname:port in the endpoint URL
 //
 // bucket: specifies the bucket on the nats server for all of the k3s values for this cluster (optional)
 //
-// context: specifies the nats context to load from ~/.config/nats/context/ e.g. nats-context for ~/.config/nats/context/nats-context.json
+// contextFile: specifies the nats context to load from ~/.config/nats/context/ e.g. nats-context for ~/.config/nats/context/nats-context.json
```

Comment:
This is kind of confusing, since it's not strictly the file, but the file to be loaded is derived from the name provided here. I think it should either remain just `contex` and automatically derive the filename of the context to load, or it should be `contextFile` and require the full path+name+extension of the context to be loaded.

Reply:
You're right... I missed updating this comment, the intent was to only support `contextFile=/some/path/context.json` and that is how it is currently implemented